docs/installation.md

```diff
@@ -19,7 +19,7 @@ The following instructions are for setting up a version of Amundsen using Docker
     If it's your first time, you may want to proactively go through [troubleshooting](troubleshooting) steps, especially the first one related to heap memory for ElasticSearch and Docker engine memory allocation (leading to Docker error 137).
 4. Ingest provided sample data into Neo4j by doing the following: _(Please skip if you are using Atlas backend)_
 
-   * In a separate terminal window, change directory to [databuilder](https://github.com/amundsen-io/amundsen/tree/main/databuilder).
+   * In a separate terminal window, change directory to the [amundsendatabuilder](https://github.com/amundsen-io/amundsendatabuilder) submodule.
```

Comment:
I think this is old reference (should be to monorepo)

Reply:
I created this `patch-2` branch off of `main` branch. Should I use `master` branch instead?

Reply:
this was changed very recently by a different PR, i'd try rebasing of freshest main

Reply:
Fixed. Sorry about the confusion and inconvenience.